google/ggrc-core
# WIP: GGRC-4317, GGRC-2138, GGRC-4794 Document epic Document object

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [ ] GGRC-4075
- [ ] GGRC-4319
- [ ] #7614

# Issue description
Upgrade document object to 1st class object.

# Steps to test the changes

To reviewers
to check all functionality please run following background tasks:

/admin/create_missing_revisions
/admin/propagate_acl
/admin/reindex
Follow user stories GGRC-2138

# Sanity checklist




# PR Review checklist
